src/docs/unbricking/unbrick-suzyq.md

```diff
@@ -64,6 +64,14 @@ Let's get to it:
 If you are not sure which file to use for your device / do not know your device's board name, you can reference [the supported devices page](supported-device.html).
 :::
 
+::: tip
+On Alderlake devices, you may not see the flash chip right away. In this case you need to do the following:
+1. `echo "apshutdown" | sudo tee -a /dev/ttyUSB2`
```

Comment:
maybe add a warning that this will shut down the chromebook? It's possible that the user may have connected the SuzyQ to the same machine.

Reply:
And normally flashing over CCD doesn't shut down the machine?

Reply:
Please excuse my own stupidity for not looking on what page we are lol